exporter/otlphttpexporter/otlp.go

```diff
@@ -124,7 +125,9 @@ func (e *exporter) pushLogs(ctx context.Context, ld plog.Logs) error {
 }
 
 func (e *exporter) export(ctx context.Context, url string, request []byte) error {
-	e.logger.Debug("Preparing to make HTTP request", zap.String("url", url))
+	var span = trace.SpanFromContext(ctx)
```

Comment:
use `trace. SpanContextFromContext()`

Reply:
all these are great feedbacks!   thank you so much!

Reply:
Fixed by the latest commit.